module3/projects/take_home/presentation_guidelines.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Take-Home Challenge Presentation Guidelines
+layout: page
+---
+
+## Presentation Guidelines
+
+This guide will serve as a general outline for how students can present their code during a take-home challenge presentation. 
+
+<section class="note">
+### Note
+
+Keep in mind that some companies will have their own guidelines on presenting which takes precenedence over this guideline.
+</section>
+
+**Prior to the presentation:**
+
+- Clear your screen of all windows or tabs except the ones relevant to your presentation. 
+
+- Have any servers fired up and ready to go.
+
+<!-- Note: Do we want them to demo in postman if there is a frontend? -->
+- Have Postman setup in advance to show your endpoints (BE).
```

Comment:
Hmmm I don't think it's necessary. Their BE presentation could instead focus on db structure, error handling, testing, REST

Reply:
Agreed^